sdcm/cluster.py

```diff
@@ -1763,6 +1763,7 @@ def install_scylla(self, scylla_repo):
                     apt-get install software-properties-common -y
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6B2BFD3660EF3F5B
                     apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 17723034C56D4B19
```

Comment:
@amoskong, why did you leave the old keys as well?

Reply:
```
We have a new key to sign deb packages, and the old releases are used in
upgrade testing. So let's reserve the old keys.
```